### PR TITLE
Initial commit PHP implode documentation

### PR DESCRIPTION
**Pull request by freshjah**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/implode/implode.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+```php
+<?php
+    $to_be_imploded = array('I','love','learning','with','Codecademy');
+    echo implode(' ',$to_be_imploded);
+?>
+```
+
+## Codebyte Example
+
+The following example show the use of `implode()` function with and without the use of the `separator`.
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/implode/implode.md:43]:
```suggestion
The following example shows the use of `implode()` function with and without the use of the `separator`.
```

File: content/php/concepts/string-functions/terms/implode/implode.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+```
+
+`implode()` accepts the following parameters:
+
+- `separator` - Optional string value. If omitted defaults to empty string.
+- `array` - An array of string elements to implode.
+
+## Example
+
+The example below uses the `implode()` function to create the sentence 'I love learning with Codecademy'. The separator is a single white space.
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/implode/implode.md:32]:
```suggestion
The example below uses the `implode()` function to create the sentence 'I love learning with Codecademy'. The separator is a single space.
```

**Review by SSwiniarski — APPROVED** [review]
@freshjah I did a first review and left a couple of comments.

**Review by SSwiniarski — APPROVED** [review]
@freshjah Looks good for a second review.

File: content/php/concepts/string-functions/terms/implode/implode.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+```codebyte/php
+<?php
+    $hyphenated = array('Merry','go','round');
+    $regular = array('Fe', 'at', 'ures');
+
+    $hyphenated_word = implode('-', $hyphenated);
+    $regular_word = implode( $regular );
+
+    echo $hyphenated_word;
+    echo("<br/>");
```
Comment (KTom101) [inline content/php/concepts/string-functions/terms/implode/implode.md:54]: I'm noticing in the codebyte for [another php entry](https://www.codecademy.com/resources/docs/php/string-functions/explode) that the `<br>` are not being rendered correctly, let's change to the new line character for now: 
```suggestion
    echo("\r\n");
```

File: content/php/concepts/string-functions/terms/implode/implode.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    echo $regular_word;
+    echo("<br/>");
```
Comment (KTom101) [inline content/php/concepts/string-functions/terms/implode/implode.md:56]:
```suggestion
    echo("\r\n");
```

File: content/php/concepts/string-functions/terms/implode/implode.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+## Syntax
+
+```pseudo
+implode($separator, $array)
+```
+
+`implode()` accepts the following parameters:
+
+- `separator` - Optional string value. If omitted defaults to empty string.
```
Comment (KTom101) [inline content/php/concepts/string-functions/terms/implode/implode.md:27]:
```suggestion
- `separator` - Optional string value. If omitted, the default is an empty string.
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@freshjah I've completed a second review of this entry.

**Comment by SSwiniarski** [comment]
In the interest of getting this merged, I implemented @KTom101's suggestions.

**Review by KTom101 — APPROVED** [review]
LGTM

**Comment by KTom101** [comment]
@freshjah Link is live: https://www.codecademy.com/resources/docs/php/string-functions/implode

Currently, you're listed as an anonymous contributor:

Screenshot 2023-05-27 at 2 12 34 PM

If you want to remain anonymous, there is nothing more you need to do. If you wish to be listed as a contributor, make sure you've added your GitHub username to your Codecademy profile using the following steps:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

YourGitHubName

2.) Then log out and log back into Codecademy using your GitHub login:

ReLogIn

> **Note:** Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to properly work. If you use the same email for both GitHub and Codecademy already, then you're set.

cc: @yangc95 @SSwiniarski

**Pull request merged**